pycbc/workflow/pegasus_sites.py

```diff
@@ -27,7 +27,6 @@
     sing_version = version
 else:
     sing_version = last_release
-sing_version = '2.3.9'
```

Comment:
Can you briefly explain why the last release was broken?

Reply:
I pressed the wrong button and released from the main branch, not the release branch. This change here would have meant we used the v2.3.9 singularity image, but that wouldn't have mattered; we've not changed the inspiral code. Fixing this here anyway.

Reply:
We have changed the inspiral code - the switch I put in between compressed / uncompressed waveforms in the case that they aren't in the bank

Reply:
Ah, then this change is *definitely* needed!